packages/ui/src/components/va-menu-list/VaMenuList.stories.ts

```diff
@@ -0,0 +1,162 @@
+import { VaDivider, VaButton, VaDropdown, VaIcon, VaAvatar } from '../../components'
+import VaMenuList from './VaMenuList.vue'
+import VaMenuItem from './components/VaMenuItem.vue'
+import VaMenuGroup from './components/VaMenuGroup.vue'
+import { addText } from '../../../.storybook/interaction-utils/addText'
+
+export default {
+  title: 'VaMenuList',
+  component: VaMenuList,
+  tags: ['autodocs'],
+}
+
+export const Default = () => ({
+  components: { VaMenuList },
+  data: () => ({
+    options: ['Option 1', 'Option 2', 'Option 3'],
+  }),
+  template: '<VaMenuList :options="options" />',
+})
+
+export const IconSlot = () => ({
+  components: { VaMenuList, VaAvatar },
+  data: () => ({
+    options: ['Option 1', 'Option 2', 'Option 3'],
+  }),
+  template: `
+  <VaMenuList :options="options">
+    <template #left-icon>
+      [left-icon]
+    </template>
+    <template #icon>
+      [icon]
+    </template>
+  </VaMenuList>
+  `,
+})
+addText(
+  IconSlot,
+  'There is no slot for right icon, only for left one.',
+  'broken',
+)
+
+const longGroupName = 'This is a very long group name that should be truncated'.repeat(10)
+export const Groups = () => ({
+  components: { VaMenuList, VaAvatar },
+  data: () => ({
+    options: [
+      { id: '0', text: 'one', value: 'one', icon: 'accessible_forward', rightIcon: '', group: '' },
+      { id: '1', text: 'two', value: 'two', icon: '', rightIcon: 'home', group: longGroupName },
+      { id: '2', text: 'three', value: 'three', icon: 'accessible_forward', rightIcon: '', group: longGroupName },
+      { id: '3', text: 'four', value: 'four', icon: '', rightIcon: 'accessible_forward', group: 'A2' },
+      { id: '4', text: 'five', value: 'five', icon: 'search', rightIcon: '', group: 'A2' },
+    ],
+    value: false,
+  }),
+  template: `
+  <VaMenuList :options="options">
+  </VaMenuList>
+  `,
+})
+addText(
+  Groups,
+  'Long group name is not truncated in story. We also have too much options - should be minimal amount.',
+  'broken',
+)
+
+export const SlotUsage = () => ({
+  components: { VaMenuList, VaMenuItem, VaMenuGroup },
+  template: `
+    <VaMenuList>
+      <VaMenuGroup group-name="Group 1" />
+      <VaMenuItem>
+        User 1
+      </VaMenuItem>
+      <VaMenuItem>
+        User 2
+      </VaMenuItem>
+      <VaMenuGroup group-name="Group 2" />
+      <VaMenuItem rightIcon="home">
+        User 3
+      </VaMenuItem>
+    </VaMenuList>
+  `,
+})
+
+export const Color = () => ({
+  components: { VaMenuList, VaMenuItem, VaMenuGroup },
+  data: () => ({
+    options: ['Option 1', 'Option 2', 'Option 3'],
+  }),
+  template: `
+    <VaMenuList color="warning" :options="options" />
+  `,
+})
+addText(
+  Groups,
+  `I don't think we have any other component where 'color' means 'hover-color', let's make it explicit.` ,
```

Comment:
Color means component accent color in general. Color doesn't mean hover-color, otherwise hover color would be exactly primary (hover color have less opacity). In case you need to change hover-color I'll add a css variable for that.

Reply:
![image](https://github.com/epicmaxco/vuestic-ui/assets/5394573/78b9aaf4-666c-45ca-b28c-c75f524be2f0)

Just to double check we're on the same page - this is a story for color. And this color is set on hover.

So color doesn't create any coloration by default - it requires hover. Which I believe is never how color works for other components. I would expect font color to change or background or something, but not hover color.

Reply:
I checked a couple of other frameworks - and they don't have color for menu at all. So how about we remove it and only keep variable?